src/App.css

```diff
@@ -1,3 +1,22 @@
 .App {
   text-align: center;
 }
```

Comment:
This file shouldn't be changed.  They should be exact as it is in main branch. Please reverse them to original.